tests/core/data/layer/v1/common/test_repository.py

```diff
@@ -0,0 +1,14 @@
+"""
+Test config files
+"""
+
+import unittest
+
+from py4ai.core.data.layer.v1.common.repository import QueryOptions
+
+
+class TestConfig(unittest.TestCase):
```

Comment:
You should use `py4ai.core.tests.core.TestCase` instead of `unittest.TestCase`

Reply:
Done